src/DatePicker/CalendarHeader.jsx

```diff
@@ -14,6 +14,7 @@ export const CalendarHeader = (props) => {
 
   const selectNextMonth = () => onMonthChange(currentMonth.clone().add(1, 'months'));
   const selectPreviousMonth = () => onMonthChange(currentMonth.clone().subtract(1, 'months'));
+  const weekdays = [0, 1, 2, 3, 4, 5, 6].map(dow => moment().weekday(dow).format('dd'));
```

Comment:
Please change variable name to english word.

Reply:
@dmtrKovalenko sorry for the stupid question, but where's the "non-english" part? are you refering to dow which should be dayOfWeek in your opinion?

Reply:
Oh, yep i have misslead the dow with foreign word meaning "day", please can you change it to dayOfWeek for example.
Thanks